features/developer_portal/admin/account/payment_details.feature

```diff
@@ -4,7 +4,16 @@ Feature: Dev Portal Buyer Payment Details
   I want to enter my billing and credit card details and keep them up to date
 
   Background:
-    Given a buyer logged in to a provider
+    Given a provider
+    And the default product of the provider has name "My API"
+    And the following application plan:
+      | Product | Name | Default | Cost per month |
+      | My API  | Gold | true    | 100            |
+    And an approved buyer "John" signed up to the provider
+    And the following application:
+      | Buyer | Name   |
+      | John  | My App |
+    And the buyer logs in to the provider
```

Comment:
Again this "buyer logs in to the provider"... what does this even mean?... :cry:

Reply:
How about this @josemigallas :point_up: ?

Seems straightforward, we just rename the step to make it clearer (at least it will make sense)